src/data/prefixes.js

```diff
@@ -0,0 +1,60 @@
+const knex = require("../knex");
+const config = require("../cfg");
+
+/**
+ * @param {String} userId
+ * @returns {Promise<string>}
+ */
+ async function getPrefix(userId) {
+  if (! config.allowPrefix || userId == null) return "";
+  const prefixRow = await knex("name_prefixes")
+    .where("user_id", userId)
+    .first();
+
+  return prefixRow ? prefixRow.prefix : "";
```

Comment:
```suggestion
  return prefixRow?.prefix : "";
```

Reply:
Note that optional chaining is not supported on Node.js 12, which modmail supports.

Reply:
Since the .nvmrc file and the prerequisites only mention Node.js 14, I just assumed that was the only supported version